projects/curaengine.cmake

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2022 Ultimaker B.V.
+# cura-build is released under the terms of the AGPLv3 or higher.
+
+GetFromEnvironmentOrCache(
+		NAME
```

Comment:
This file is a mix of tab-indented and space-indented. For our code style we have to indent everything with 4 spaces per level.

Reply:
Yeah somehow my IDE did some weirdness in a few files. Will look into it